### ISSUE TITLE
Do I run docker-compose on server or on client?

### ISSUE THREAD

**Issue by krisk0** [hassiweb/docker-open5gs#2]
I have docker server on one computer and docker client on another (Gentoo Linux). 

I attempted to follow README.md, sitting on the client. I came to the point of **docker-compose**. It refused to start. Investigation showed that it failed to bind-mount config directory.

I wouild like to know, if it is at all possible to run docker-compose on computer not running docker daemon? If yes, how do I get bind-mounting to work?

**Comment by hassiweb** [comment]
Do you mean that you did not use the docker-ce package to install docker?
At the least, container runtime (e.g., containerd) must be required to run containers.

Also, which directory did you fail to mount?
If you did not change the directory structure of this repository, there is a possibility that there would be some changes in terms of the directory structure in the latest version of Open5GS.  I have not tried the latest version.  So in this case, you have to fix it or you can also use the old version of Open5GS.
Or if it is the PGW, there is a possibility of the TUN device.  The PGW requires TUN to have GTP tunnels for UEs's connections.  In this case, please confirm the TUN device is available in your environment.

**Comment by krisk0** [comment]
I installed docker with **emerge app-emulation/docker**  where emerge is gentoo'ish apt and app-emulation/docker is a name of package which contains both client and server. containerd is provided by **app-emulation/containerd**, and got automagically installed because app-emulation/containerd is a dependency of app-emulation/docker.

The problem is that mounting of config directory silently fails, therefore all containers started by your docker-compose.yaml  don't do anything useful. I changed entrypoint command from "/bin/sh" "/etc/open5gs/run_hss.sh" to "sleep" "1h", attached to the running hss container, did **ls /etc/open5gs**, the directory was empty.

Bind mount systematically fails on computer not running dockerd server. For instance, theese commands
```
mkdir unique_dir
cd unique_dir
>got.file
docker run --rm -it -v `pwd`:/please.mount ubuntu ls -l /please.mount
```
print **total 0** when run on client computer, when unique_dir does not exist on server.

I used docker before (approximately 2 years ago) with dedicated dockerd server and did not experience problems with mounts.

I would like you to test bind-mounting on non-server computer. If that works, I would like to know your docker and linux kernel version.
